packages/@sanity/default-layout/src/update/PortableTextContent/PortableTextContent.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import {PortableText, PortableTextComponents} from '@portabletext/react'
+import {Box, Card, Code, Heading, Stack, Text} from '@sanity/ui'
+import urlBuilder from '@sanity/image-url'
+import {getProject} from '@sanity/asset-utils'
+import {ImageCard, ListBox, ListBoxOuter, RootBox, SpaceBox} from './PortableTextContent.styles'
+
+interface PortableTextContentProps {
+  value: any
+}
+
+function Image({value}) {
```

Comment:
...and if typed, we'd get the right type here - `MaterializedImage` (or whatever we want to call it)